pages/resources/status.vue

```diff
@@ -101,18 +100,20 @@
                       <td class="status-badge">
                         <a
                           :href="
-                            repo.versions.length > 1
-                              ? 'https://github.com/hapijs/' + repo.name + '/branches/actions?query=workflow%3Aci'
-                              : 'https://github.com/hapijs/' + repo.name + '/actions?query=workflow%3Aci'
+                             version.license === 'BSD'
```

Comment:
What is the significance of the BSD license?

Reply:
I believe that's what @lloydbenson uses to differentiate hapi versions between open-source license versions and the commercial ones Eran manages.

Reply:
Yep it was the cleanest way to determine the non commercial branches which don't have valid ci status. Othewise you have to do things like if its not hapi but ... kind of logic